Formula/git-secret.rb

```diff
@@ -0,0 +1,27 @@
+class GitSecret < Formula
+  desc "Bash-tool to store the private data inside a git repo."
+  homepage "https://sobolevn.github.io/git-secret/"
+
+  url "https://github.com/sobolevn/git-secret/archive/v0.1.2.tar.gz"
+  sha256 "96032c1c0b2b161c4df5b8627be8d98a97a92a5d32127c9a5e8686bd64b5a3e7"
+  head "https://github.com/sobolevn/git-secret.git"
+
+  bottle :unneeded
+  depends_on "gnupg" => :recommended
+
+  def install
+    system "make", "build"
+    system "bash", "utils/install.sh", prefix
+  end
+
+  def caveats
+    <<-EOS.undent
+      You now have the `git secret` command! Find out how to use it
+      by typing `man git-secret` or visiting 'https://sobolevn.github.io/git-secret/'.
+    EOS
+  end
+
+  test do
+    system "git", "secret", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!